src/app/components/register-card/register-card.component.scss

```diff
@@ -0,0 +1,59 @@
+.register-card-wrapper {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    height: 100%;
+}
+
+.register-card-container {
+    display: flex;
+    flex-direction: column;
+    width: 500px;
+    border-left-color: #E92B8E;
+}
+
+.register-card-header {
+    display: flex;
+    justify-content: center;
+    color: #E92B8E;
+}
+
+.register-card-content {
+    display: flex;
+    justify-content: center;
```

Comment:
display: flex;
 justify-content: center;
 
 this is repeated a lot. define a class for it in the styles.scss and use that class instead of defining so many classes with the same style.
 
 .flex-container-center {
    display: flex;
    justify-content: center;
 }